[2025-05-25T08:15:27.697+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-05-25T08:15:27.712+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: supplier_data_etl_pipeline.extract_sql_data scheduled__2025-05-24T00:00:00+00:00 [queued]>
[2025-05-25T08:15:27.723+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: supplier_data_etl_pipeline.extract_sql_data scheduled__2025-05-24T00:00:00+00:00 [queued]>
[2025-05-25T08:15:27.724+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 6
[2025-05-25T08:15:27.742+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): extract_sql_data> on 2025-05-24 00:00:00+00:00
[2025-05-25T08:15:27.748+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=497) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-25T08:15:27.750+0000] {standard_task_runner.py:63} INFO - Started process 506 to run task
[2025-05-25T08:15:27.749+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'supplier_data_etl_pipeline', 'extract_sql_data', 'scheduled__2025-05-24T00:00:00+00:00', '--job-id', '189', '--raw', '--subdir', 'DAGS_FOLDER/supplier_dag.py', '--cfg-path', '/tmp/tmpzilsk_ru']
[2025-05-25T08:15:27.751+0000] {standard_task_runner.py:91} INFO - Job 189: Subtask extract_sql_data
[2025-05-25T08:15:27.780+0000] {task_command.py:426} INFO - Running <TaskInstance: supplier_data_etl_pipeline.extract_sql_data scheduled__2025-05-24T00:00:00+00:00 [running]> on host a7d211f7b093
[2025-05-25T08:15:27.836+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='supplier_data_etl_pipeline' AIRFLOW_CTX_TASK_ID='extract_sql_data' AIRFLOW_CTX_EXECUTION_DATE='2025-05-24T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-24T00:00:00+00:00'
[2025-05-25T08:15:27.838+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-05-25T08:15:27.839+0000] {etl.py:78} INFO - connected to db source
[2025-05-25T08:15:27.851+0000] {etl.py:80} INFO - queried db source
[2025-05-25T08:15:27.852+0000] {etl.py:82} INFO - closed db connection
[2025-05-25T08:15:27.854+0000] {etl.py:84} INFO - db source pickled
[2025-05-25T08:15:27.855+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-05-25T08:15:27.855+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-05-25T08:15:27.861+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=supplier_data_etl_pipeline, task_id=extract_sql_data, execution_date=20250524T000000, start_date=20250525T081527, end_date=20250525T081527
[2025-05-25T08:15:27.888+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-05-25T08:15:27.906+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-25T08:15:27.907+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
